tmt/steps/execute/internal.py

```diff
@@ -73,15 +76,17 @@ def options(cls, how=None):
             help='Disable interactive progress bar showing the current test.'))
         return options + super().options(how)
 
-    def wake(self, keys=None):
+    # TODO: use better types once superclass gains its annotations
+    def wake(self, keys: Optional[List[str]] = None) -> None:
         """ Wake up the plugin, process data, apply options """
         super().wake(keys=keys)
         # Make sure that script is a list
         tmt.utils.listify(self.data, keys=['script'])
 
     # TODO: consider switching to utils.updatable_message() - might need more
     # work, since use of _show_progress is split over several methods.
-    def _show_progress(self, progress, test_name, finish=False):
+    def _show_progress(self, progress: Any, test_name: str,
```

Comment:
Follow what code calls `_show_progress` and with what arguments, and you will find out `progress` can have a more strict type than `Any`, because AFAICT, it's `str`.